CharacterClient/main.py

```
@@ -0,0 +1,66 @@
+import argparse
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)